torch/_inductor/ir.py

```diff
@@ -3627,6 +3627,10 @@ def __init__(
             tuple(tensor_args),
             tuple(nontensor_args),
         )
+        # We need output buffers for generating kernel arguments in the
+        # abi-compatible mode, where we retrieve outputs by pass each individual
+        # output through the abi-compatible interface.
+        self.outputs = []
```

Comment:
Is there a reason this can't be populated at construction time?

Reply:
Hmm, it is populated at construction time (in the classmethod ```create```):

https://github.com/pytorch/pytorch/pull/110003/files#diff-cf6ca00beddc32a2a6a2933fb9913b6a2b925ffc3b745488967210e4343134acR3886

Or, you were referring to another point?

Reply:
no, I understand what this is doing now